dev-libs/libical/libical-2.0.0.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="An implementation of iCalendar protocols and data formats"
+DESCRIPTION="Libical is an Open Source implementation of the iCalendar \
+protocols and protocol data units. The iCalendar specification describes how \
+calendar clients can communicate with calendar servers so users can store \
+their calendar data and arrange meetings with other users."
+HOMEPAGE="http://libical.github.io/libical/"
+COPYRIGHT="Eric Busboom"
+LICENSE="MPL v1.0
+	GNU LGPL v2.1"
+REVISION="1"
+scrGitRev="6c4af23b0a95fd105f38e879908cbc80390f3716"
+SOURCE_URI="https://github.com/libical/libical/releases/download/v$portVersion/libical-$portVersion.tar.gz"
+CHECKSUM_SHA256="654c11f759c19237be39f6ad401d917e5a05f36f1736385ed958e60cf21456da"
+SOURCE_DIR="libical-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	libical$secondaryArchSuffix = $portVersion
+	lib:libical$secondaryArchSuffix
```

Comment:
According to Gentoo and Arch Linux the sonames for all these libs in release 2.0.0 are:
```
libical.so.2.0.0
libical_cxx.so.2.0.0
libicalss.so.2.0.0
libicalss_cxx.so.2.0.0
libicalvcal.so.2.0.0
```
I guess we could therefore use "= $portVersion compat >= 2" for all the **`lib:libical*`** lines in PROVIDES and **`devel:libical*`** lines in BUILD_REQUIRES. Would that be OK to use that? My understanding is that we should avoid this in the general case, but since upstream is using these sonames in sync with the version string, it would be cool to use $portVersion here.

BTW, maybe we should drop **`libical-2.1.0~git.recipe`** and only keep **`libical-2.0.0.recipe`**.